flaskApi/file_db.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import file_system_db_util as util
+
+
+class Filedb:
+    def __init__(self):
+        # get dir to file folder (create if dont exist)
+        root_dir = os.path.dirname(os.getcwd())
+        self.UPLOAD_DIR = os.path.join(root_dir, 'files', 'uploads')
```

Comment:
This is not an actuall const if you implement it that way - https://stackoverflow.com/questions/10672419/class-constants-in-python